src/Classes/Container.ts

```diff
@@ -27,7 +35,7 @@ export class Container {
     public bubble: IBubble;
     public repulse: IRepulse;
     public svg: ISvg;
-    public img: IImage;
+    public image: IImage;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Declaration of public instance field not allowed after declaration of private static method. Instead, this should come at the beginning of the class/interface.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5036765)